src/Class.js

```diff
@@ -1,10 +1,10 @@
 var WebJack = {};
 
-(function(exports){
```

Comment:
This stuff was to make it usable directly in a browser (non-npm) -- where there may not be an `exports` object available. Is this a better way to do it? Just checking!